packages/core/src/useNavigationBuilder.tsx

```diff
@@ -20,8 +20,13 @@ import {
   Router,
   RouterFactory,
   PartialState,
+  PrivateValueStore,
 } from './types';
 
+// This is to make TypeScript compiler happy
+// eslint-disable-next-line babel/no-unused-expressions
+PrivateValueStore;
```

Comment:
Is this some kind of a circular dep? Maybe mark as a TODO to fix later

Reply:
Nope not circular. Probably TS complains because this type is not exported and all properties on it are private.